carma-messenger-core/cpp_message/test/test_MobilityRequest.cpp

```diff
@@ -14,12 +14,13 @@
+    std::vector<uint8_t> binary_input={0,240,129,225,77,90,113,39,212,90,209,171,22,12,38,173,56,147,234,45,104,213,131,150,172,88,65,133,14,36,88,204,88,177,98,197,139,22,43,89,50,100,201,107,54,108,217,173,131,6,12,21,172,88,177,98,197,139,22,44,88,177,98,229,147,38,108,219,178,96,205,179,134,173,27,183,106,225,131,126,116,60,60,182,225,95,67,102,30,155,247,238,211,187,58,131,36,193,149,6,137,131,42,13,83,6,84,27,49,100,201,155,54,236,152,51,108,225,171,70,237,218,184,96,221,91,237,167,117,253,221,118,226,203,201,211,24,213,101,71,145,18,179,8,115,33,80,112,225,155,69,155,52,227,203,187,158,91,252,54,97,233,149,212,73,245,22,177,96,193,131,53,152,240,242,229,167,47,43,251,176,237,202,234,158,157,157,178,242,65,83,151,92,122,208,70,145,94,10,10,145,164,82,134,179,30,30,92,180,229,229,127,78,71,85,105,196,159,81,3,6,12,24,48,96,197,103,124,186,115,232,232,233,102,28,156,239,243,223,155,167,124,60,178,223,237,151,151,61,59,247,58,167,231,159,76,187,80,86,203,203,158,157,251,144,85,221,175,118,254,251,150,100,195,211,45,253,249,175,236,195,207,165,254,125,48,244,203,127,78,238,124,50,227,233,167,126,231,86,108,217,178,182,108,213,177,34,44,201,135,166,91,251,243,95,217,135,159,75,248,114,115,191,143,14,205,56,185,97,233,167,126,231,86,108,217,178,182,108,213,177,34,44,225,203,45,254,156,180,240,191,135,39,59,250,50,225,217,211,69,252,122,50,227,214,234,63,44,185,119,44,195,147,157,254,125,48,244,235,205,213,44,185,22,105,231,206,255,60,123,249,101,116,209,202,206,25,121,109,211,210,255,44,188,122,233,229,151,35,166,11,58,105,219,151,159,76,59,120,58,98,213,195,87,12,219,55,102,197,195,22,137,131,42,14,83,6,84,26,166,12,168,54,114,193,147,54,109,217,48,102,217,195,86,141,219,181,112,193,184,79,171,234,250,190,207,179,236,197,171,38,108,219,178,96,205,179,134,173,27,183,106,225,131,112};
```

Comment:
Wow!  I have no idea how to verify this.  It is definitely different.  I will trust it is okay.